server/database/queries/getAllEvents.js

```diff
@@ -1,7 +1,11 @@
 
 const dbconnection = require('../db_connection')
 const events = () => {
+  const sql = {
```

Comment:
not a big deal but I don't see any benefit in wrapping the SQL in this object, IMO it would be better directly in the `dbconnection.query()` call